public/activities/imspackagepainter.js

```diff
@@ -276,39 +278,39 @@ var ImsPackagePainter = {
 		Simva.getActivityResultForUser(activity, user, function(error, result){
 			if(error){
 				$.toast({
-					heading: 'Error loading the result',
+					heading: `Error loading the result`,
 					text: error.message,
-					position: 'top-right',
-					icon: 'error',
+					position: `top-right`,
+					icon: `error`,
 					stack: false
```

Comment:
It is wise to use single quotes by default and only, if required by use case, use backticks to allow interpolation

Reply:
fixed